### PR TITLE
Updated version number

### PR DESCRIPTION
**Pull request by akhenry**
Updates the version number in package.json in master preparation to tag the last pre-tcr version of master.

## Author Checklist

* Changes address original issue? Y
* Unit tests included and/or updated with changes? N/A
* Command line build passes? Y
* Changes have been smoke-tested? Y

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "openmct",
-  "version": "0.14.0-SNAPSHOT",
+  "version": "0.14.0",
```
Comment (psarram) [inline package.json]: @akhenry Shouldn't this be v0.15.0?

Comment (akhenry) [reply package.json]: Last Open MCT release was [v0.13.3](https://github.com/nasa/openmct/releases/tag/v0.13.3). [The process](https://github.com/nasa/openmct/blob/v0.13.3/docs/src/process/version.md#version-numbering)  is that the version number is the next release with '-SNAPSHOT' appended to it. When that version is released, the `-SNAPSHOT` is removed, and the release is tagged. It is then updated with the next version number and a '-SNAPSHOT'.

Comment (psarram) [reply package.json]: The reason I asked was the reference to 0.15.0 version number in the README. That file needs to be updated then.

Comment (akhenry) [reply package.json]: Good catch. Fixed in https://github.com/nasa/openmct/pull/2575

**Comment by psarram** [comment]
**Reviewer Checklist**

- Changes appear to address issue? Yes
- Appropriate unit tests included? N/A
- Code style and in-line documentation are appropriate? Yes
- Commit messages meet standards? Yes

**Pull request merged**